sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -33,6 +33,10 @@
 #include "modules/computer_vision/viewvideo.h"
 #include "modules/computer_vision/cv.h"
 
+#include "modules/computer_vision/video_thread.h"
+
+#include "boards/bebop.h"
```

Comment:
Strange this is added here, would suggest that this only works with the bebop. Is there anyway you can make this more general? Does this also work on the ARDrone?